guides/source/association_basics.md

```diff
@@ -1519,20 +1519,30 @@ conditions exists in the collection. It uses the same syntax and options as
 
 ##### `collection.build(attributes = {}, ...)`
 
-The `collection.build` method returns one or more new objects of the associated type. These objects will be instantiated from the passed attributes, and the link through their foreign key will be created, but the associated objects will _not_ yet be saved.
+The `collection.build` method returns a single or array of new objects of the associated type. The object(s) will be instantiated from the passed attributes, and the link through their foreign key will be created, but the associated objects will _not_ yet be saved.
```

Comment:
Suggestions:
- "a single object or an array of"
- "of the associated type"
- no comma after "attributes"

"The `collection.build` method returns a single object or an array of new objects of the associated type. The object(s) will be instantiated with the passed attributes and the link through their foreign key will be created, but the associated objects will _not_ yet be saved."